templates/zerver/help/change-a-user's-name.md

```diff
@@ -0,0 +1,25 @@
+# Change a user's name
+Name of the user can be changed by the admin as well as user himself.
+
+Following are the steps to change the name of the user using an admin account:
+* Click on the ![cog](/static/images/help/cog.png) icon in the upper right corner
```

Comment:
Use `Click on the cog (![cog](/static/images/help/cog.png))` like the other documentation we have.